### PR TITLE
Add select option

### PR DESCRIPTION
**Pull request by clayscode**
Adds the ability to select columns rather than just discard. Helpful if working with a big list of columns. 
Screenshot 2023-11-22 at 2 13 00 PM

### CODE REVIEW

File: lib/assets/data_transform_cell/main.js [hunk 1]
```diff
@@ -682,6 +695,7 @@ export async function init(ctx, payload) {
             </div>
             <div class="add-operation">
               <BaseButton label="Discard" @add-operation="addOperation('discard')" :disabled="noDataFrame || noAvailableData"/>
+              <BaseButton label="Select" @add-operation="addOperation('select')" :disabled="noDataFrame || noAvailableData"/>
```
Comment (cristineguadelupe) [inline lib/assets/data_transform_cell/main.js:698]: We are keeping the buttons in alphabetical order!

File: lib/assets/data_transform_cell/main.css [hunk 2]
```diff
@@ -55,6 +55,10 @@
   --beige-100: #FBF5F1;
   --beige-200: #F8EBE4;
   --beige-600: #EDCDBB;
+
+  --indigo-100: #e0e7ff;
```
Comment (cristineguadelupe) [inline lib/assets/data_transform_cell/main.css:59]: We're using colors that don't conflict with the Livebook palette. I think both blue and green are good choices for the `Select` and you can base it on the current `Fill missing` or` Filter by` colors.

**Review by cristineguadelupe — CHANGES_REQUESTED** [review]
Thanks for your contribution @clayscode. Just two small tweaks!

File: lib/assets/data_transform_cell/main.css (hunk 2 above)
Comment (clayscode) [reply lib/assets/data_transform_cell/main.css:59]: Screen Shot 2023-11-24 at 3 33 57 PM

How about emerald?

**Review by cristineguadelupe — APPROVED** [review]

**Pull request merged**